src/components/Urlsnarf/Urlsnarf.tsx

```diff
@@ -1,44 +1,57 @@
-    "Step 2: Input the Interface.\n" +
-    "       Eg: eth0\n\n" +
-    "Step 3: Enter any Exclusion details within the sniff.\n" +
-    "       Eg: POST (every packet besides POST will be shown)\n\n" +
-    "Step 4: Click Sniff to commence Urlsnarf's operation.\n\n" +
-    "Step 5: View the Output block below to view the results of the tool's execution.";
-
+/**
+ * Represents the form values for the Urlsnarf component.
+ */
 interface FormValuesType {
     listenerInputType: string;
     listenerArgs: string;
     versusMode: string;
 }
 
-const listeners = ["Interface", "Packet capture file"];
-
+/**
+ * Renders the Urlsnarf component.
+ * @returns The rendered Urlsnarf component.
+ */
 const Urlsnarf = () => {
-    const [loading, setLoading] = useState(false);
-    const [output, setOutput] = useState("");
-    const [selectedListenerInput, setSelectedListenerInput] = useState("");
-    const [pid, setPid] = useState("");
-    const [allowSave, setAllowSave] = useState(false);
-    const [hasSaved, setHasSaved] = useState(false);
-
+    // Component State Variables.
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
+    const [selectedListenerInput, setSelectedListenerInput] = useState(""); // State variable to store the selected listener input.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving the output.
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if the output has been saved.
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+
+    // Component constants.
+    const listeners = ["Interface", "Packet capture file"]; // List of listener settings.
+    const isListenerInterface = selectedListenerInput === "Interface"; // Check if the selected listener input is an interface.
+    const isListenerFile = selectedListenerInput === "Packet capture file"; // Check if the selected listener input is a file.
+    const title = "Urlsnarf"; // Title of the component.
+    const description =
+        "Urlsnarf outputs all requested URLs sniffed from HTTP traffic in Common Log Format, used by almost all web servers."; // Description of the component.
+    const steps =
+        "Step 1: Select the Listener settings.\n" +
+        "Step 2: Input the Interface.\n" +
+        "Step 3: Enter any Exclusion details within the sniff.\n" +
+        "Step 4: Click Sniff to start Urlsnarf.\n" +
```

Comment:
Change so it matches the suggested conventional button text. 
```suggestion
        "Step 4: Click the start Urlsnarf button.\n" +
```